### PR TITLE
Add description of embedded Auth in Image in Prezi

### LINKED ISSUE #526: Image auth from Presentation
Lots of use cases for caching authentication requirements in manifests / collections.

### PR DESCRIPTION
**Pull request by azaroth42**
Closes #526

(Sorry for lack of preview links, plane wifi is too slow to manually deploy)

### CODE REVIEW

**Comment by mikeapp** [comment]
:+1:

**Comment by zimeon** [comment]
:+1:

**Comment by azaroth42** [comment]
@jpstroop @tomcrane ?

File: source/api/presentation/2.1/index.md [hunk 1]
```diff
@@ -1499,6 +1499,7 @@ Recipes for enabling CORS and the conditional Content-type header are provided i
 
 It may be necessary to restrict access to the descriptions made available via the Presentation API.  As the primary means of interaction with the descriptions is by web browsers using XmlHttpRequests across domains, there are some considerations regarding the most appropriate methods for authenticating users and authorizing their access.  The approach taken is described in the [Authentication][auth] specification, and requires requesting a token to add to the requests to identify the user.  This token might also be used for other requests defined by other APIs.
 
+It is possible to include the links to Image API services within the Manifest and other Presentation API responses, and thus it is also possible to include within those, links to Authentication API services.  The first time an Authentication service is included, it _MUST_ be the complete description. Subsequent references _SHOULD_ be just the URL of the service, and clients are expected to look up the details from the full description given this URL.  Clients _MUST_ anticipate situations where the Authentication service description in the manifest is out of date: the source of truth is the Image Information document.
```
Comment (jpstroop) [inline source/api/presentation/2.1/index.md]: First sentence makes no sense to me

Comment (azaroth42) [inline source/api/presentation/2.1/index.md]: Rewriting ...

**Comment by jpstroop** [comment]
:+1:

**Comment by zimeon** [comment]
:+1: , I thought I understood it before but this is much better

**Pull request merged**